src/main/java/keycontacts/model/student/StudentComparator.java

```diff
@@ -10,7 +10,16 @@
  * field
  */
 public class StudentComparator implements Comparator<Student> {
-
+    public static final String NAME_ASCENDING = "Name (ascending)";
```

Comment:
I think we could consider naming this as NAME_ASCENDING_MESSAGE_PART or something like that to make it clearer, cuz it var name NAME_ASCENDING could look like a dedicated enum or type.